src/pytest/__init__.py

```diff
@@ -89,6 +90,7 @@
 
 
 __all__ = [
+    "HIDDEN_PARAM",
```

Comment:
Can you move this so the list is kept sorted?

Reply:
Formatter forces `ALL_CAPS` names to the top of the list

Reply:
Didn't realize that, thanks